manifests/java.yml

```diff
@@ -1453,14 +1453,8 @@ tests/:
         spring-boot-3-native: missing_feature (GraalVM. Tracing support only)
     test_user_blocking_full_denylist.py:
       Test_UserBlocking_FullDenylist:
-        '*': v0.111.0
-        jersey-grizzly2: v1.7.0
-        ratpack: v1.7.0
-        resteasy-netty3: v1.7.0
-        spring-boot: v0.110.0
+        '*': v1.37.0
```

Comment:
This is effectively a no-op because the corresponding test file has the following annotation:
```
@bug(library="java", reason="APPSEC-56006")
```

The problem here is probably not the manifest, but that the annotation needs to be  updated.

Reply:
Missed this, thanks.
Fixed

Reply:
@ValentinZakharov So, it's one or the other: either we have the old versions here in this manifest + the annotation for bug, or we have the version here. Let's discuss offline if needed.